packages/datatrak-web/src/components/Button.tsx

```diff
@@ -2,30 +2,54 @@
  * Tupaia
  *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
-import React, { ReactNode, Fragment } from 'react';
+import React, { ReactElement, ReactNode } from 'react';
 import { To, Link as RouterLink } from 'react-router-dom';
-import { Tooltip, Button as UIButton } from '@tupaia/ui-components';
+import { Tooltip as BaseTooltip, Button as UIButton } from '@tupaia/ui-components';
 import styled from 'styled-components';
 
+const TOOLTIP_COLOR = '#002d47';
 const StyledButton = styled(UIButton)`
   &.Mui-disabled {
     pointer-events: auto; // this is to allow the hover effect of a tooltip to work
   }
 `;
 
+const Tooltip = styled(BaseTooltip).attrs({
+  disablePortal: true,
+})`
+  .MuiTooltip-tooltip {
+    background-color: ${TOOLTIP_COLOR};
+    .MuiTooltip-arrow {
+      color: ${TOOLTIP_COLOR};
+    }
+  }
+`;
 interface ButtonProps extends Record<string, any> {
   tooltip?: string;
   children?: ReactNode;
   to?: To;
 }
-export const Button = ({ tooltip, children, to, ...restOfProps }: ButtonProps) => {
-  const Wrapper = tooltip ? Tooltip : Fragment;
 
+const ButtonWrapper = ({
+  children,
+  tooltip,
+}: {
+  children: ReactElement<any, any>;
+  tooltip?: ButtonProps['tooltip'];
+}) => {
+  if (!tooltip) return children;
+  return (
+    <Tooltip title={tooltip} arrow>
+      {children}
+    </Tooltip>
+  );
+};
```

Comment:
This was to stop an error in the console about `Fragment` not being allowed to have a `title` prop